static/js/cohort_details.js

```diff
@@ -327,13 +326,7 @@ require([
         return false;
     });
 
-    // If this is a new cohort, set TCGA Project selected as default
```

Comment:
Woops! No, definitely not removed on purpose. A victim of an overzealous bugfix. Want me to do another request with this fixed?